sql/addons_view_v1.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
+  @submission_date as submission_date,
+  document_id,
+  client_id,
+  sample_id,
+  subsession_start_date,
+  normalized_channel,
+  addons_array.element.*
+FROM
+  telemetry.main_summary_v4
+CROSS JOIN UNNEST(active_addons.list) as addons_array
```

Comment:
This should use the following form, for consistency with existing queries in the repo:

```sql
FROM
  telemetry.main_summary_v4,
  UNNEST(active_addons.list) AS _addon
```